client/components/cancel/cancellationSummary.tsx

```diff
@@ -175,14 +183,18 @@ export const isCancelled = (subscription: Subscription) =>
 	Object.keys(subscription).length === 0 || subscription.cancelledAt;
 
 export const getCancellationSummary =
-	(productType: ProductType, productFriendlyName: string) =>
+	(productType: ProductType, cancelledProductDetail: ProductDetail) =>
 	(productDetail: ProductDetail) =>
 		isCancelled(productDetail.subscription) ? (
-			actuallyCancelled(productType, productDetail, productFriendlyName)
+			actuallyCancelled(
+				productType,
+				productDetail,
+				cancelledProductDetail,
+			)
 		) : (
 			<GenericErrorScreen
 				loggingMessage={
-					productFriendlyName +
+					productType.friendlyName(cancelledProductDetail) +
```

Comment:
really minor, but this could be a template literal:

`${productType.friendlyName(cancelledProductDetail)} cancellation call succeeded but subsequent product detail doesn't show as cancelled`